smarty/forcefield_utils.py

```diff
@@ -364,6 +364,50 @@ def getMolParamIDToAtomIndex( oemol, ff):
 
     return param_usage
 
+def merge_system( topology0, topology1, system0, system1, label0="AMBER system", label1 = "SMIRFF system", verbose = True):
+    """Merge two given OpenMM systems. Returns the merged OpenMM System.
+
+    Parameters
+    ----------
+    topology0 : OpenMM Topology
+        Topology of first system (i.e. a protein)
+    topology1 : OpenMM Topology
+        Topology of second system (i.e. a ligand)
+    system0 : OpenMM System
+        First system for comparison (usually from AMBER)
+    system1 : OpenMM System
+        Second system for comparison (usually from SMIRFF)
+    label0 (optional) : str
+        String labeling system0 for output. Default, "AMBER system"
+    label1 (optional) : str
+        String labeling system1 for output. Default, "SMIRFF system"
+    verbose (optional) : bool
+        Print out info on energies, True/False (default True)
+
+    Returns
+    ----------
+    topology : OpenMM Topology
+    system : OpenMM System
+    positions: ???
```

Comment:
I wasn't sure if the function should or shouldn't return positions. Does it need to?